contributing.md

```diff
@@ -0,0 +1,46 @@
+
+## Code of Conduct
+Please be nice to each other when interacting with others and follow our [code of conduct](<link to code of conduct>).
+
+## How to contribute
+In order to contribute you can either add a new issue or you can create a pull request and link it to an existing issue. If the code adds a feature that is not already present in an issue, you can create a new issue for the feature and add the pull request to it.
+
+### Contributing by adding an issue
+If you have found a bug and would like to report it or if you have a feature that you feel we should add, then it would be appreciated if you contributed by adding an issue. Before you do so please search the other issues to ensure that you are not attempting to add an issue that has already been created. In order to submit a new issue just hit the issue button and a choice between two templates should appear and just follow along with the template you chose. If you don't know how to fill in all parts of the template just go ahead and skip those parts. You can edit the issue later.
+
+### Contributing through code
+In order to contribute through code follow the steps below. Note that you don't need to be the best programmer to contribute. Our discord is open for questions
+
+ 1. Pick a feature you would like to add or a bug you would like to fix. 
+	 - If you wish to contribute but what you want to fix/add is not already listed in the issues go ahead and add the issue
+ 2. Create a fork of the repository
+	 - A fork creates your own version of the code on your Github, so you can work on it separately from everyone else.
+	 - You can learn more about forking [here](https://docs.github.com/en/get-started/quickstart/fork-a-repo)
+ 3. Ensure that you have commit signing enabled
+	 - This ensures that the code you submit is made by you and not someone else say that they are you
+	 - You can learn more about how to setup commit signing [here](https://www.freecodecamp.org/news/what-is-commit-signing-in-git/ "https://www.freecodecamp.org/news/what-is-commit-signing-in-git/")
+	 - If you have already made some commits that you wish to put in a pull request without signing them, then you can follow [this guide](https://dev.to/jmarhee/signing-existing-commits-with-gpg-5b58) on how to fix that.
+ 5. Clone the repository to your your computer.
+	 - This puts a copy of your fork of the code on your computer so you can edit it
+	 - You can learn more about cloning repositories [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+ 6. Build the project
+	 - For a detailed look on how to build {{project-name}} look at our [README file](<link to readme>)
+ 7. Now is the time to start writing the code you wanted
+	 - Just fire up your favorite code editor and make the changes that you wanted to make to the repository
```

Comment:
> Just fire up your favorite code editor and make the changes that you wanted to make to the repository

Love the energy here, but it could be intimidating to folks who are newer to programming. I think the "Just" could be interpreted as "it's so easy!". Being a bit nitpicky here and maybe reading too much into it.

One alternative way you could open this sentence is with "Open your favorite code editor...".